x-pack/plugins/cases/common/api/cases/case.ts

```diff
@@ -258,7 +258,6 @@ export const CasesFindRequestRt = rt.exact(
     /**
      * The field to use for sorting the found objects.
      *
```

Comment:
would be nice to have somewhere documenting which fields are supported though

Reply:
Yes, we can do it when we apply the limits in 8.10. The types will give us the supported fields.